experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/lazyField.spec.ts

```diff
@@ -78,3 +114,321 @@ describe("lazyField", () => {
 		);
 	});
 });
+
+/**
+ * Initializes a test tree, context, and cursor, and moves the cursor to the tree's root.
+ *
+ * @returns The initialized context and cursor.
+ */
+function initializeTreeWithContent<Kind extends FieldKind, Types extends AllowedTypes>(
+	schema: TypedSchemaCollection,
+	initialTree?:
+		| SchemaAware.TypedField<FieldSchema, SchemaAware.ApiMode.Flexible>
+		| readonly ITreeCursorSynchronous[]
+		| ITreeCursorSynchronous,
```

Comment:
```suggestion
	content: TreeContent
```

Reply:
If you instead want to try and make this all more type safe, use TreeContent's type paramater.

If you wanted to make it more type safe the way it was, you could have used TypedSchemaCollection's type parameter.

Reply:
actually, since the return value has no extra type info, adding type arguments here doesn't really add any value.